src/diracx/db/jobs/db.py

```diff
@@ -1,16 +1,18 @@
 from __future__ import annotations
 
+import logging
 from datetime import datetime, timezone
 from typing import Any
 
-from sqlalchemy import func, insert, select, update
+from sqlalchemy import delete, func, insert, select, update
 
 from diracx.core.exceptions import InvalidQueryError
+from diracx.core.models import ScalarSearchOperator, ScalarSearchSpec
 from diracx.core.utils import JobStatus
 
 from ..utils import BaseDB, apply_search_filters
 from .schema import Base as JobDBBase
-from .schema import InputData, JobJDLs, Jobs
+from .schema import InputData, JobJDLs, JobParameters, Jobs, OptimizerParameters
```

Comment:
In https://github.com/DIRACGrid/diracx/issues/29 (and at the last hackathon) we discuss against adding MySQL support for `JobParameters`. Before we ratify this decision I would, at a minimum, avoid adding this here.

Reply:
Also, I am TBH not sure that `OptimizerParameters` will be something that we'll need in DiracX. So, avoid adding code for it atm.